src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -125,9 +125,9 @@ public function start(): bool
          * The part `[a-zA-Z0-9,-]` is related to the PHP ini directive `session.sid_bits_per_character` defined as 6.
          * See https://www.php.net/manual/en/session.configuration.php#ini.session.sid-bits-per-character.
          * Allowed values are integers such as:
-         * - 4 for range `a-f0-9`
+         * - 4 for range `a-f0-9` (@deprecated since Symfony 7.2, only 5 bits per character will be allowed in Symfony 8.0)
          * - 5 for range `a-v0-9`
```

Comment:
```suggestion
         * - 4 for range `a-f0-9`
         * - 5 for range `a-v0-9` (@deprecated since PHP 8.4)
```

Looking at the PR in PHP only 4 is allowed. 
https://github.com/php/php-src/pull/15213/files#diff-22481788aee01357f8e628c9191632d127a78a2ea20cfbf9c3354cdc420d4ff2R777

Reply:
Indeed, the RFC was not super clear about the default value. Thanks!